.coderabbit.yaml

```
@@ -0,0 +1,130 @@
+# CodeRabbit configuration for WStunnel project
+# https://docs.coderabbit.ai/guides/configure-coderabbit
+
+# Language-specific settings
+language: "en-US"
+
+# Review settings
+reviews:
+  # Request reviews on pull requests
+  request_changes_workflow: false
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)